app/services/mapping_code_determiner.rb

```diff
@@ -53,8 +53,14 @@ def dispute_type_mapping_code
   def case_type_mapping_code
     # TODO: Add further when-branches once we have additional case types
     case tribunal_case.case_type
+    when CaseType::APPLY_DECISION_ENQUIRY
+      MappingCode::APPN_DECISION_ENQRY
```

Comment:
FWIW, the only place these constants should be used is in here, which is the intersection between the "GLiMR language" and "our language".